README.md

```diff
@@ -33,6 +33,42 @@ When a proposal has taken enough shape that it warrants detailed feedback and it
     1. The PR may include links to supporting materials that could not be included with the proposal files, such as white papers. Add links to the "Supporting Materials" section of the PR body.
     2. Please mention and link any issues and PRs in [OpenUSD](https://github.com/PixarAnimationStudios/OpenUSD) or [OpenUSD-proposals](https://github.com/PixarAnimationStudios/OpenUSD-proposals) that are related to the new proposal. A label will be created to link relevant discussions together. 
 
+### Guidance for Writing Proposals
+
+When writing and submitting a proposal, we make the following suggestions for receiving the best feedback.
+
+1. Include a link to the rendered proposal in your PR description. This can be as simple as linking to the markdown file in your source branch.
+   
+   This makes it significantly easier for someone to read the document in a well formatted way.
+2. Your PR and proposal should include, and ideally start with, a short summary of what your change would achieve.
+
+   A possible structure could be:
+   1. **Summary** of what you're hoping to achieve
+   2. A **problem statement** explaining why what you are proposing is not currently possible
+   3. A **glossary of terms** that readers may not be familiar with.
+   4. If applicable, any links to **existing implementations or reference documents** that may be useful 
+   5. **Details** about your proposal, such as why you are making certain choices
+   6. **Risks** that you anticipate (if any)
+   7. **Alternate solutions** that you have considered (if any), and why you didn't go with them
+   8. Any **excluded topics**, that you have left out and why. These may be things you want to handle separately in the
+      future for example.
+
+3. It is highly recommended that any submitted proposal include text examples of what your proposal would look like
+   in `.usda` syntax.
+4. It's recommended that long sentences be split over multiple lines in your Markdown file.
+   This allows more granular feedback, and easier viewing of the files. A recommended line length is 120 characters.
```

Comment:
```suggestion
   This allows more granular feedback, and easier viewing of the files.
```
I suggest we don't recommend line lengths. Doesn't seem worth opening a debate about "holy" line lengths which specifying 120 would seem to beg.

Reply:
Removed!